packaging/nnstreamer.spec

```diff
@@ -391,6 +391,9 @@ export NNSTREAMER_DECODERS=$(pwd)/build/ext/nnstreamer/tensor_decoder
     LD_LIBRARY_PATH=./tests/nnstreamer_filter_edgetpu:. ./tests/nnstreamer_filter_edgetpu/unittest_edgetpu --gst-plugin-path=. --gtest_output="xml:unittest_edgetpu.xml"
     popd
 %endif #ifarch 64
+    pushd build
```

Comment:
As ncsdk2 package is defined only under `%{with tizen}`, how about moving this test case under the same condition or on line 383?

Reply:
@kparichay Sounds great. I updated it in V3.